app/controllers/users_controller.rb

```diff
@@ -0,0 +1,69 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[show edit update destroy]
+
+  # GET /users or /users.json
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1 or /users/1.json
+  def show; end
+
+  # GET /users/new
+  def new
+    @user = User.new
+  end
+
+  # GET /users/1/edit
+  def edit; end
```

Comment:
[ OPTIONAL]
You can consider remvoving actions not used from your controllers. This makes your codebase clean and professional.